hw/cxl/cxl-host.c

```diff
@@ -248,26 +247,27 @@ static MemTxResult cxl_write_cfmws(void *opaque, hwaddr addr,
 
     d = cxl_cfmws_find_device(fw, addr);
     if (d == NULL) {
+        trace_cxl_debug_message("CXL device not found");
         /* Writes to invalid address are silent */
         return result;
     }
-    type = object_get_typename(OBJECT(d));
-
-    g_assert(addr < 0x8000000);
```

Comment:
Same here. Why remove assert?

Reply:
Not sure why these two lines disappeared - added back.